multi_folder_config/provider_settings.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This is ADAS specific configuration, it should not be in this repository. 
This is typically the kind of thing which has its place in the ansible repository, and can then be mounted in the container.

Reply:
Done in the next commit.